src/main/java/betterquesting/commands/admin/QuestCommandDefaults.java

```diff
@@ -230,7 +229,8 @@ public static void save(@Nullable ICommandSender sender, String databaseName, Fi
                 return;
             }
 
-            NBTTagCompound questTag = quest.writeToNBT(new NBTTagCompound());
+            NBTTagCompound questTag = quest.writeToNBT(new NBTTagCompound(), true);
+            questTag.setInteger("questID", questId);
```

Comment:
this has the same reason as the `lineID` and `order` properties, but its merely a missing value from the file that ought to be there instead of causing a bug.